### PR TITLE
[zh]sync admin task files

### PR DESCRIPTION
**Pull request by chenxuc**
related: #29329

sync Chinese files for following:
(11) admin 6
1	1	content/zh/docs/tasks/administer-cluster/limit-storage-consumption.md
1	1	content/zh/docs/tasks/administer-cluster/change-pv-reclaim-policy.md (already synced by others)
17	9	content/zh/docs/tasks/administer-cluster/controller-manager-leader-migration.md
14	1	content/zh/docs/tasks/administer-cluster/cpu-management-policies.md
1	1	content/zh/docs/tasks/administer-cluster/access-cluster-api.md

### CODE REVIEW

File: content/zh/docs/tasks/administer-cluster/access-cluster-api.md [hunk 1]
```diff
@@ -300,10 +300,10 @@ func main() {
 ```
 
 <!--
-If the application is deployed as a Pod in the cluster, please refer to the [next section](#accessing-the-api-from-a-pod).
+If the application is deployed as a Pod in the cluster, see [Accessing the API from within a Pod](/zh/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod).
 -->
 如果该应用程序部署为集群中的一个
-Pod，请参阅[下一节](#accessing-the-api-from-within-accessing-the-api-from-within-a-pod)。
+Pod，请参阅[从 Pod 内访问 API](/docs/tasks/access-application-cluster/access-cluster/#accessing-the-api-from-a-pod)。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/access-cluster-api.md:306]: zh 前缀加错位置了。303 和 306

File: content/zh/docs/tasks/administer-cluster/controller-manager-leader-migration.md [hunk 2]
```diff
@@ -64,14 +61,14 @@ If you use a tool to administrator the cluster, please refer to the documentatio
...
-Although it is possible to migrate within the same version, ideally the migration should be performed as part of a upgrade so that changes of configuration can be aligned to releases. 
+Although it is possible to migrate within the same version, ideally the migration should be performed as part of a upgrade so that changes of configuration can be aligned to each release. 
 The exact versions of N and N + 1 depend on each cloud provider. For example, if a cloud provider builds a `cloud-controller-manager` to work with Kubernetes 1.22, then N can be 1.21 and N + 1 can be 1.22.
 
 The control plane nodes should run `kube-controller-manager` with Leader Election enabled through `--leader-elect=true`. 
 As of version N, an in-tree cloud privider must be set with `--cloud-provider` flag and `cloud-controller-manager` should not yet be deployed.
 -->
 假定控制平面正在运行 Kubernetes N 版本，并且要升级到 N+1 版本。
-尽管可以在同一版本中进行迁移，但理想情况下，迁移应作为升级的一部分执行，以便可以更改配置与发布保持一致。
+尽管可以在同一版本中进行迁移，但理想情况下，迁移应作为升级的一部分执行，以便可以更改配置与每个发布保持一致。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/controller-manager-leader-migration.md:71]:
```suggestion
尽管可以在同一版本中进行迁移，但理想情况下，迁移应作为升级的一部分执行，以便可以更改配置与每个发布版本保持一致。
```

File: content/zh/docs/tasks/administer-cluster/controller-manager-leader-migration.md [hunk 3]
```diff
@@ -80,19 +77,21 @@ N 和 N+1的确切版本取决于各个云驱动。例如，如果云驱动构
...
 
-For authorization, this guide assumes that the cluser uses RBAC. 
+For authorization, this guide assumes that the cluster uses RBAC. 
 If another authorization mode grants permissions to `kube-controller-manager` and `cloud-controller-manager` components, 
 please grant the needed access in a way that matches the mode.
 -->
 树外云驱动必须已经构建了一个实现领导者迁移的 `cloud-controller-manager`。
 如果云驱动导入了 v0.21.0 或更高版本的 `k8s.io/cloud-provider` 和 `k8s.io/controller-manager`，
 则可以进行领导者迁移。
+但是，对 v0.22.0 以下的版本，领导者迁移是一项 Alpha 阶段功能，它需要特性门控 `ControllerManagerLeaderMigration` 来启用。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/controller-manager-leader-migration.md:94]:
```suggestion
但是，对 v0.22.0 以下的版本，领导者迁移是一项 Alpha 阶段功能，它需要启用特性门控 
`ControllerManagerLeaderMigration`。
```

File: content/zh/docs/tasks/administer-cluster/cpu-management-policies.md [hunk 4]
```diff
@@ -310,3 +319,30 @@ equal to one. The `nginx` container is granted 2 exclusive CPUs.
...
+如果使用 `full-pcpus-only` 策略选项，static 策略总是会分配完整的物理核心。
+你可以通过在 CPUManager 策略选项里加上 `full-pcups-only=true` 来启用该选项。
+<!--
+By default, without this option, the static policy allocates CPUs using a topology-aware best-fit allocation.
+On SMT enabled systems, the policy can allocate individual virtual cores, which correspond to hardware threads.
+This can lead to different containers sharing the same physical cores; this behaviour in turn contributes
+to the [noisy neighbours problem](https://en.wikipedia.org/wiki/Cloud_computing_issues#Performance_interference_and_noisy_neighbors).
+-->
+默认情况下，如果不使用该选项，static 策略会使用拓扑感知最适合的分配方法来分配 CPU。
+在启用了 SMT 的系统，策略会分配对应硬件线程的单独虚拟核心。
```
Comment (tengqm) [inline content/zh/docs/tasks/administer-cluster/cpu-management-policies.md:339]:
```suggestion
在启用了 SMT 的系统上，此策略所分配是与硬件线程对应的、独立的虚拟核。
```

**Comment by chenxuc** [comment]
@tengqm Thanks for the review. I've updated the doc with your comments.

**Comment by tengqm** [comment]
Many thanks.
/approve

**Comment by ydFu** [comment]
/lgtm
Thank you for your contribution

**Pull request merged**